src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationHandler.cs

```diff
@@ -0,0 +1,169 @@
+﻿using JetBrains.Annotations;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Text;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace AspNet.Security.OAuth.Alipay
+{
+    /// <summary>
+    /// 支付宝 OAUTH2.0 网站登录
```

Comment:
Either remove this or use comments like the below. This sort of information is not useful for Intellisense comments in an IDE.

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/1a32506eedbacd83fed18b985aa11d58928192d7/src/AspNet.Security.OAuth.Amazon/AmazonAuthenticationHandler.cs#L22-L24